_posts/2019-04-16-sicko-mode.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "\"Sicko\" Mode: a modern critique of homophobia in America"
+author: Michael Luo
+date:   2019-04-16
+categories: drake travis scott
+album_art: assets/images/astroworld.png
+---
+
+A less careful listener may simply dismiss Travis Scott and Aubrey “Drake”
+Graham's hit song *Sicko Mode* as a mere modern pop trap record, perhaps noting
+the song’s strange choice of three different beats before requesting a change
+of song to Shack Wes' *Mo Bamba*. The risk of such negligent listening is to
+miss the song’s true message: the story of a young man exploring his sexuality.
+
+<iframe allow="autoplay *; encrypted-media *;"
+style="width:100%;max-width:660px;overflow:hidden;background:transparent;"
+sandbox="allow-forms allow-popups allow-same-origin allow-scripts
+allow-storage-access-by-user-activation
+allow-top-navigation-by-user-activation"
+src="https://embed.music.apple.com/
+us/album/sicko-mode/1421241217?i=1421242781&app=music"
+height="150" frameborder="0"></iframe>
+
+The first beat of the song is mild and unassuming - a careful allegory chosen
+to represent our society’s heteronormativity. When our protagonist is in the
+stage this song describes (i.e. has still not come out), by obeying the
+heterosexual norms of society, he is repressed just like the beat of the song.
+Aubrey “Drake” Graham (hereby solely referred to as Drake), plays a close
+friend of our protagonist. He notes that as for heterosexuality, “that shit all
+he know, he don’t know nothin’ else”. Unable to see our hero living a lie,
+our artists take action by awakening the protaganist's true sexuality (“I
+tried to show him, yea I tried to show him”). Their attempt is successful, as
+they note our character is “goin’ on you with the pick and roll”. For readers
+unfamiliar with basketball, a pick and roll involves a player forcing a
+defender to *switch* to defending him by setting up a screen. These words are
+accompanied by a beat switch -- and also accompanied by our character’s switch
+of sexuality.
+
+The second beat is more aggressive and powerful, consistent with our
+character’s newfound interest in masculinity.
+
+> Different coloured chains, think my jeweller really sellin’ fruits
+
+ He immediately affirms his new sexuality by buying multicoloured jewellery,
```

Comment:
Is this spacing intentional?

Reply:
Also jewellery is spelled wrong

Reply:
Jewellery is British English. Valid.

Reply:
Please address the extraneous spacing at the beginning of the line.